alpha/ui.cpp

```diff
@@ -19,6 +20,11 @@ void _TS_UI::staticTask(void* parameter)
 
 void _TS_UI::begin()
 {
+
+  this->buttonA = new IOButton(BUTTONA);
```

Comment:
can all the IOButton things be in/part of HAL? all IOs should be in or beneath HAL

Reply:
Yup sure I'll move this instantiating to HAL